index.js

```diff
@@ -112,11 +113,13 @@ async function runMediasoupWorkers()
 
 async function createExpressApp()
 {
-	logger.info('creating Express app...');
+	logger.info(`Booting...${process.env.DEV ? 'DEV' : 'LIVE'}`);
 
 	expressApp = express();
 	expressApp.use(bodyParser.json());
-
+	expressApp.use(cors({
+		origin: process.env.DEV ? /\.githubpreview\.dev$/ : /\.webaverse\.com$/
```

Comment:
Should be careful here, this might make the app undevelopable locally, which is done through things like `local.webaverse.com`

Reply:
I'm actually fine whitelisting all domains for now.

Reply:
Roger, I'll get that changed